cpu/esp32/Kconfig

```diff
@@ -16,6 +16,17 @@ config HAS_CPU_ESP32
     help
         Indicates that the current CPU belongs to the ESP32x SoC series.
 
+config HAS_ESP_BLE
+    bool
+    help
+        Indicates that the used ESP32x SoC supports Bluetooth LE.
+
+config HAS_ESP_BLE_ESP32
+    bool
+    help
+        Indicates that the ESP32x SoC uses the SDK Bluetooth LE library
+        for the ESP32 variant.
```

Comment:
Would it be possible that only one of those two would be true, while the other would be false? If not, maybe only one symbol is enough?

Reply:
> Would it be possible that only one of those two would

Yes. Feature `HAS_ESP_BLE` only indicates that the SoC has BLE while `HAS_ESP_BLE_ESP32` indicates which library package is used since each ESP32x variant has its own one. For ESP32-C3, there will be `HAS_ESP_BLE_ESP32C3`
```
config HAS_ESP_BLE_ESP32C3
    bool
    help
        Indicates that the ESP32x SoC uses the SDK Bluetooth LE library
        for the ESP32-C3 variant.
```
We could drop these additional symbol if `PACKAGE_ESP32_SDK_LIB_BT_ESP32*` depend on `CPU_FAM_*`
```diff
  config PACKAGE_ESP32_SDK_LIB_BT_ESP32
      bool "ESP32 SDK Bluetooth library for the ESP32 SoC"
      depends on TEST_KCONFIG
      depends on HAS_ARCH_ESP32
-     depends on HAS_ESP_BLE_ESP32
+     depends on CPU_FAM_ESP32
      help
          Vendor SDK Bluetooth library for ESP32 SoC
```
but I don't know whether this is would be compliant with Kconfig rules.

Reply:
The dependencies in `cpu/esp32/Makefile.dep` look already like this
```make
  ifeq (esp32,$(CPU_FAM))
    FEATURES_REQUIRED += esp_ble_esp32
    USEPKG += esp32_sdk_lib_bt_esp32
  else ifeq (esp32c3,$(CPU_FAM))
    FEATURES_REQUIRED += esp_ble_esp32c3
    USEPKG += esp32_sdk_lib_bt_esp32c3
  endif
```

Reply:
OK, thanks for the explanation :)